modules/core/src/main/java/org/apache/ignite/internal/processors/query/QueryUtils.java

```diff
@@ -1610,6 +1638,23 @@ public static SqlFieldsQuery withQueryTimeout(SqlFieldsQuery qry, int timeout, T
         return qry;
     }
 
+    /**
+     * @return {@code True} if output sensitive data allowed.
+     */
+    public static boolean includeSensitive() {
+        return INCLUDE_SENSITIVE || INCLUDE_SENSITIVE_TL.get();
+    }
+
+    /**
+     * @return Delimeter to use.
+     */
+    public static char delimeter() {
```

Comment:
it`s unclear why sensitive logic affect delimiter behavior.

Reply:
By default, AST SQL output is multiline.
So user queries will look ugly in `SYS.SQL_QUERIES_HISTORY`, JMX and other tools that outputs SQL query history.